### DISCUSSION TITLE
Getting Error while running the first Docker Compose Command

### DISCUSSION THREAD

**Discussion by dhruv050992** [aspnetrun/run-aspnetcore-microservices#19] [Q&A]
Getting below error while running  
"docker-compose -f docker-compose.yml -f docker-compose.override.yml up -d" command

Error
 => ERROR [build 5/8] RUN dotnet restore "Services/Catalog/Catalog.API/Ca  2.8s
------
 > [build 5/8] RUN dotnet restore "Services/Catalog/Catalog.API/Catalog.API.csproj":
#13 1.549   Determining projects to restore...
#13 2.350 /usr/share/dotnet/sdk/5.0.202/NuGet.targets(131,5): error : Unable to load the service index for source https://api.nuget.org/v3/index.json. [/src/Services/Catalog/Catalog.API/Catalog.API.csproj]
#13 2.350 /usr/share/dotnet/sdk/5.0.202/NuGet.targets(131,5): error :   The SSL connection could not be established, see inner exception. [/src/Services/Catalog/Catalog.API/Catalog.API.csproj]
#13 2.350 /usr/share/dotnet/sdk/5.0.202/NuGet.targets(131,5): error :   The remote certificate is invalid because of errors in the certificate chain: PartialChain [/src/Services/Catalog/Catalog.API/Catalog.API.csproj]
------
ERROR: Service 'catalog.api' failed to build

**Comment by mehmetozkaya** [comment]
It seems related like microsoft base images. 
Could you please try with changing FROM parts into Dockerfile of Catalog.API as below images and try again. I will also try from my part.

FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
...
FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build

**Reply by dhruv050992** [reply]
I have tried this, It's not working.

Actually, I am working in proxy environment, Its blocking the content to be downloaded from Nuget.

**Reply by mehmetozkaya** [reply]
yes that would be problem, so that means, in that environment you can't connect Nuget. We should fix it first.

**Reply by dhruv050992** [reply]
I have tried many things but no success. Please can you help me?

**Reply by dhruv050992** [reply]
I am able to use Nuget for my windows but whenever it connects to docker, I am not able to use Nuget.

**Reply by mehmetozkaya** [reply]
Actually I can't find the solution yet, let me investigate further and try to find a proper solution.
